assets/js/role_slider.js

```diff
@@ -0,0 +1,12 @@
+// const roleSliderContainer = document.getElementById("roleSlider");
+// const roleSliderTabs = document.querySelectorAll(".tab");
+// roleSliderTabs[0].children[0].classList.add('selected');
+
+// roleSliderContainer.addEventListener('mousedown', (e) => {
+//     roleSliderTabs.forEach(tab => {
+//         tab.children[0].classList.remove('selected');
+//         if (tab.contains(e.target)) {
+//             tab.children[0].classList.add('selected');
+//         };
+//     });
+// });
```

Comment:
One last nit on this, I swear!

If the css-only setup worked, would you mind removing this file (and from the associated head partial)?

Reply:
Oh shoot I forgot to remove this! My bad!

Reply:
@nonowazu fixed in commit https://github.com/The-Balance-FFXIV/glam/pull/63/commits/f9fbce0568dc5a840fde1d8445d24742fc61b1d3